pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerDe.scala

```diff
@@ -282,3 +282,9 @@ object SerDe {
     value.foreach(v => writeBytes(out, v))
   }
 }
+
+object SerializationFormats {
```

Comment:
Yeah I don't sealed traits will do be sufficient here as we need the string value to be passed to R. Sealed traits would be good to use if all usages are within Scala